README.md

```diff
@@ -1,5 +1,8 @@
 # SPDX.jl
 
+[![GitHub Actions](https://github.com/SamuraiAku/SPDX.jl/workflows/CI/badge.svg)](https://github.com/SamuraiAku/SPDX.jl/actions/workflows/CI.yml)
+[![codecov](https://codecov.io/gh/SamuraiAku/SPDX.jl/branch/master/graph/badge.svg?token=TODO)](https://codecov.io/gh/SamuraiAku/SPDX.jl)
```

Comment:
Once you've granted codecov access to this repo, you'll be able to replace this "TODO" with the token found at https://app.codecov.io/gh/SamuraiAku/SPDX.jl/settings/badge